src/router/backoffice.js

```diff
@@ -40,6 +40,24 @@ const routes = {
           /* webpackChunkName: "backoffice" */ "../views/backoffice/healthsnapshots/HealthSnapshotList.vue"
         )
     },
+    {
+      path: "healthsnapshots/filtered",
+      name: "backoffice.healthSnapshotFiltered",
```

Comment:
You don't need to duplicate it. Just do `props: true` on the original one and make the property optional (cf. below).